package.json

```diff
@@ -6,17 +6,17 @@
   "license": "MIT",
   "main": "src/index.js",
   "scripts": {
-    "test": "mocha test/*.js test/block-management test/raw-transactions test/reducers test/transport test/utils",
-    "lint": "eslint src test/*.js test/block-management test/raw-transactions test/reducers test/transport test/utils",
+    "test": "mocha test",
+    "lint": "eslint src test",
     "coverage": "istanbul cover --include-all-sources --root ./src ./node_modules/mocha/bin/_mocha test/",
     "link": "npm link speedomatic ethereumjs-blockstream",
     "genversion": "genversion src/version.js",
     "build": "mkdirp dist && browserify ./exports.js -o ./dist/ethrpc.js && uglifyjs ./dist/ethrpc.js -o ./dist/ethrpc.min.js",
-    "build:tests": "browserify ./test/reducers/*.js -o ./test/browser/bundle.js",
     "postinstall": "postinstall-build --only-as-dependency dist",
     "prepublishOnly": "postinstall-build dist",
     "preversion": "npm test",
-    "version": "npm run genversion && git add src/version.js"
+    "version": "npm run genversion && git add src/version.js",
+    "testwhy": "wtfnode ./node_modules/mocha/bin/_mocha --timeout 5000 ./test/stub-node-only.js"
```

Comment:
This could use some documentation (a comment somewhere indicating what this does, why it exists, etc.)

Reply:
That's cruft left over from my testing actually, I'll delete it.